app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+import shutil
+
+
+def copy_file(command: str) -> None | str:
+    command_list = command.split()
+    if command_list[0] == "cp" and len(command_list) == 3:
+        _, first_file, second_file = command_list
+    else:
+        return f"Wrong command line please double check it: {command}"
```

Comment:
You can raise exception here with custom message
Same for other return statements